mappings/net/minecraft/block/GrowingPlantBlock.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_ymdmcjuw net/minecraft/block/GrowingPlantBlock
+	METHOD m_evmdilba copyState (Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_txtbiemp;
+		ARG 1 from
+		ARG 2 to
+	METHOD m_pxnibpqz getStemHeadPos (Lnet/minecraft/unmapped/C_peaveboq;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_mmxmpdoq;)Ljava/util/Optional;
```

Comment:
is this the top block? I think there could be a more straightforward name here

Reply:
It is
How about just `getTop`?

Reply:
sure!